### ISSUE TITLE
Unable to start manager in cluster (dockerized)

### ISSUE THREAD

**Issue by y0d4a** [wazuh/wazuh-docker#1267]
Hi, because of this i cannot start up and running cluster from docker compose:
https://github.com/wazuh/wazuh-docker/blob/e8d2463d9901ce279b1fcdee6ec65b860e34765e/multi-node/config/wazuh_cluster/wazuh_manager.conf#L104

manager says:

> 2024/03/18 14:24:31 wazuh-csyslogd: ERROR: (1230): Invalid element in the configuration: 'indexer'.
2024/03/18 14:24:31 wazuh-csyslogd: ERROR: (1202): Configuration error at 'etc/ossec.conf'.
2024/03/18 14:24:31 wazuh-csyslogd: CRITICAL: (1202): Configuration error at 'etc/ossec.conf'.
wazuh-csyslogd: Configuration error. Exiting

**Comment by rauldpm** [comment]
Hello @y0d4a 

The `indexer` block is a new block introduced in `4.8.0` which is still in the testing phase, please make sure you are using a repository tag and not a branch, specifically, to use the latest published version you should refer to the `v4.7.3` tag

- https://github.com/wazuh/wazuh-docker/tree/v4.7.3

I also recommend that you read the documentation on docker deployment

- https://documentation.wazuh.com/current/deployment-options/docker/wazuh-container.html

```
git clone https://github.com/wazuh/wazuh-docker.git -b v4.7.3
```

**Comment by y0d4a** [comment]
oh.... :(
Thank you for help!

Now all is fine.
